src/brewHandler.h

```diff
@@ -10,67 +10,67 @@
-boolean brewSwitchWasOff = false;
+int brewSwitchState                  = kBrewSwitchIdle;
+boolean brewSwitchWasOff             = false;
 
-double totalBrewTime = 0;                           // total brewtime set in software
-double timeBrewed = 0;                              // total brewed time
-double lastBrewTimeMillis = 0;                      // for shottimer delay after disarmed button
-double lastBrewTime = 0 ;
-unsigned long startingTime = 0;                     // start time of brew
-boolean brewPIDDisabled = false;                    // is PID disabled for delay after brew has started?
+double totalBrewTime       = 0;     // total brewtime set in software
+double timeBrewed          = 0;     // total brewed time
+double lastBrewTimeMillis  = 0;     // for shottimer delay after disarmed button
+double lastBrewTime        = 0;
+unsigned long startingTime = 0;     // start time of brew
+boolean brewPIDDisabled    = false; // is PID disabled for delay after brew has started?
 
 // Shot timer with or without scale
 #if FEATURE_SCALE == 1
-    boolean scaleCalibrationOn = 0;
-    boolean scaleTareOn = 0;
-    int shottimerCounter = 10 ;
-    float calibrationValue = SCALE_CALIBRATION_FACTOR;  // use calibration example to get value
-    float weight = 0;                                   // value from HX711
-    float weightPreBrew = 0;                            // value of scale before wrew started
-    float weightBrew = 0;                               // weight value of brew
-    float scaleDelayValue = 2.5;                        // value in gramm that takes still flows onto the scale after brew is stopped
-    bool scaleFailure = false;
-    const unsigned long intervalWeight = 200;           // weight scale
-    unsigned long previousMillisScale;                  // initialisation at the end of init()
-    HX711_ADC LoadCell(PIN_HXDAT, PIN_HXCLK);
-
-    #if SCALE_TYPE == 0 
-        HX711_ADC LoadCell2(PIN_HXDAT2, PIN_HXCLK);
-    #endif 
+boolean scaleCalibrationOn         = 0;
+boolean scaleTareOn                = 0;
+int shottimerCounter               = 10;
+float calibrationValue             = SCALE_CALIBRATION_FACTOR; // use calibration example to get value
+float weight                       = 0;                        // value from HX711
+float weightPreBrew                = 0;                        // value of scale before wrew started
+float weightBrew                   = 0;                        // weight value of brew
+float scaleDelayValue              = 2.5;                      // value in gramm that takes still flows onto the scale after brew is stopped
+bool scaleFailure                  = false;
+const unsigned long intervalWeight = 200;                      // weight scale
+unsigned long previousMillisScale;                             // initialisation at the end of init()
+HX711_ADC LoadCell(PIN_HXDAT, PIN_HXCLK);
+
+    #if SCALE_TYPE == 0
+HX711_ADC LoadCell2(PIN_HXDAT2, PIN_HXCLK);
+    #endif
```

Comment:
okay look like if we decide to do line alignment for comments, this also will be the case with stuff likes this

Reply:
OK, this is weird. 
The "canonical" way to align preprocessor macros is no indentation at all. I thought clang-tidy would refuse to do anything else :)

Reply:
I tried very hard to mimic your current style of using `#if` macros. I don't like it either and would rather fully left-align macros. If they are used as code, we should just change them for proper C++ code

```cpp
if (SCALE_TYPE == 0) {
    // ...
}
```

then it will also be aligned normally. I would do this in a separate step though. Opinions?

Reply:
fully left aligne for #if, and we get them out of the code soon

Reply:
Done, pushed.